flepimop/gempyor_pkg/tests/steps_rk4/test_fastmath_safety.py

```diff
@@ -0,0 +1,192 @@
+import numpy as np
+import pytest
+from numba import njit
+from scipy.sparse import csr_matrix
+from pathlib import Path
+import shutil
+import confuse
+
+from gempyor.model_info import ModelInfo
+from gempyor.vectorization_experiments import (
+    prod_along_axis0,
+    compute_proportion_sums_exponents,
+    compute_transition_amounts_serial,
+    compute_transition_amounts_parallel,
+    assemble_flux,
+)
+
+
+@pytest.fixture(scope="module")
+def modelinfo_from_config(tmp_path_factory):
+    tmp_path = tmp_path_factory.mktemp("model_input")
+
+    original_dir = Path(__file__).resolve()
+    while original_dir.name != "flepimop":
+        original_dir = original_dir.parent
+    tutorial_dir = original_dir.parent / "examples/tutorials"
+
+    input_dir = tmp_path / "model_input"
+    input_dir.mkdir()
+
+    input_files = [
+        "geodata_sample_2pop.csv",
+        "ic_2pop.csv",
+        "mobility_sample_2pop.csv",
+    ]
+    for fname in input_files:
+        shutil.copyfile(tutorial_dir / "model_input" / fname, input_dir / fname)
+
+    config_file = "config_sample_2pop_modifiers.yml"
+    config_path = tmp_path / config_file
+    shutil.copyfile(tutorial_dir / config_file, config_path)
+
+    config = confuse.Configuration("TestModel", __name__)
+    config.set_file(str(config_path))
+
+    return (
+        ModelInfo(
+            config=config,
+            config_filepath=str(config_path),
+            path_prefix=str(tmp_path),
+            setup_name="sample_2pop",
+            seir_modifiers_scenario="Ro_all",
+        ),
+        config,
+    )
```

Comment:
Nice use of `pytest`'s fixtures for DRY, could this be moved into `gempyor.testing` (contains testing utilities)? Maybe also could wrap `setup_example_from_tutorials` to get rid of input file setup.